tapeagents/io.py

```diff
@@ -202,3 +201,9 @@ def load_legacy_tapes(tape_class: Type[TapeType], path: Path | str, step_class:
             tape.steps = steps
         tapes.append(tape)
     return tapes
+
+
+def save_agent(agent, filename):
```

Comment:
I think this is too trivial for a helper function

Reply:
I have in mind that we will want to do agent diff from saved agents as well as agent import/export. Standard simple is a good thing, it might get more options overtime.